default_arm_joint_pos: [0, -1.57, 2.88, 0.26, 0]

min_arm_joint_pos: [-2.61,    0, -2.87, -1.51, -1.34, -2.79]
max_arm_joint_pos: [ 2.61, 2.96,     0,  1.51,  1.34,  2.79]

min_arm_joint_vel: [-1.0, -1.0, -1.0, -1.0, -1.0, -1.0]
max_arm_joint_vel: [ 1.0,  1.0,  1.0,  1.0,  1.0,  1.0]

min_body_vel: [-0.5, -0.5, -0.5]
max_body_vel: [ 0.5,  0.5,  0.5]

# 权重矩阵
W_ee_diag: [10000.0, 10000.0, 10000.0, 5000.0, 5000.0, 5000.0]  # 末端执行器权重 前三个位置后三个方向
W_vel_diag: [1000.0, 1000.0, 1000.0, 100.0, 100.0, 100.0]       # 速度权重 前三个平移后三个旋转
W_u_diag: [1, 1, 100, 100, 100, 10, 1e-2, 1e-2, 1e-2, 1e-2, 1e-2] # 控制量权重
w_x_body_pos: [100.0, 100.0, 100.0, 100.0, 100.0, 100.0]      # 机身位置权重
w_x_arm_pos: [1.0, 1.0, 1.0, 1.0, 1.0]                        # 机械臂位置权重
w_x_body_vel: [10.0, 10.0, 10.0, 10.0, 10.0, 10.0]            # 机身速度权重
w_x_arm_vel: [1.0, 1.0, 1.0, 1.0, 1.0]                        # 机械臂速度权重

# PD of Body
kpPos: [200, 200, 500]     # PD parameter of body position 
kdPos: [50, 70, 50]      
kpAng: [500, 500, 150]     # PD parameter of body angular 
kdAng: [50, 50, 50]      
kpSw: [300, 300, 300]      # PD parameter of swing leg
kdSw: [30, 30, 30]

# PD of Arm
kpEePos: [300, 300, 300]   # PD parameter of EE position
kdEePos: [30, 30, 30]
kpEeAng: [350, 350, 350]   # PD parameter of EE angular
kdEeAng: [35, 35, 35]
kpArmJ: [500, 500, 500, 500, 500]   # PD parameter of arm joint
kdArmJ: [50, 50, 50, 50, 50]


# weight of QP
wBlinear: 1
wBangle: 1
wSw: 1
wElinear: 1
wEangle: 1
wArmJ: 1